prebid_cache_client/prebid_cache.go

```diff
@@ -112,3 +126,54 @@ func Put(ctx context.Context, objs []*CacheObject) error {
 
 	return nil
 }
+
+func PutXml(ctx context.Context, objs []*CacheXmlObject) error {
```

Comment:
this function is almost identical to the Put func above. Instead of creating a new PutXml func you can reuse Put. Maybe just pass in an extra arg that tells you if it's xml or not.

Reply:
I talked with Duc about this redundant code, looks like we can't remove it... please let me know if you come up with some solution of reusing the Put func

Reply:
maybe you won't be able to reuse the entire Put function but you can probably reuse from here https://github.com/prebid/prebid-server/pull/283/files/fb9d614bcd7006fb478ae9b5204c052581cb28f0#diff-5f225f9ddabd0bcde9c281b2af14ad76R140

Reply:
That's a very good point. However, starting from the line you mentioned to the end of the Put method, we have two types of 'pr' and two types of 'objs' to attach UUID, we need to let the reusable part know every time which type (json/xml) it should to work on, which means the code will also get doubled. Instead of figuring out which is the reusable part of Put method, I would say just create the PutXml method...